dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -976,11 +975,7 @@ public String toString() {
 
   @CheckForNull
   public String getDefinitionOrDefault(ConnectorPropertyWithDefault property) {
-    String stringValue = getDefinition(property);
-    if (StringUtils.isEmpty(stringValue)) {
-      return property.getDefaultValue();
-    }
-    return stringValue;
+    return nonEmpty(getDefinition(property)).orElseGet(property::getDefaultValue);
```

Comment:
Are we sure this is easier to read?

Reply:
It depends. This one is less important compared to other changes in this PR, so I've reverted it.